packages/protocol/package.json

```diff
@@ -6,61 +6,61 @@
-    "prepare_devchain_anvil_publishing": "yarn ts-node --preferTsExts ./scripts/change-anvil-devchain-package-version.ts",
-    "validate_abis_exports": "yarn ts-node --preferTsExts ./scripts/validate-abis-package.ts",
-    "sourcify-publish": "yarn ts-node ./scripts/sourcify-publish.ts",
-    "migrate": "./scripts/bash/migrate.sh",
-    "set_block_gas_limit": "./scripts/bash/set_block_gas_limit.sh",
-    "download-artifacts": "./scripts/bash/download_artifacts.sh",
-    "init-network": "./scripts/bash/init_network.sh",
-    "upload-artifacts": "./scripts/bash/upload_artifacts.sh",
-    "upgrade": "./scripts/bash/upgrade.sh",
-    "revoke": "./scripts/bash/revoke.sh",
+    "devchain": "yarn ts-node scripts/devchain.ts",
+    "devchain:reset": "yarn devchain generate-tar .tmp/devchain.tar.gz --upto 29",
+    "ganache-dev": "./scripts/bash/ganache.sh",
+    "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
+    "generate-stabletoken-files": "yarn ts-node ./scripts/generate-stabletoken-files.ts",
     "govern": "./scripts/bash/govern.sh",
-    "console": "./scripts/bash/console.sh",
-    "check-versions": "./scripts/bash/check-versions.sh",
-    "check-opcodes": "yarn ts-node scripts/check-opcodes.ts",
+    "init-network": "./scripts/bash/init_network.sh",
+    "lint": "yarn run lint:ts && yarn run lint:sol",
+    "lint:sol": "solhint --version && solhint './contracts/**/*.sol' && solhint './contracts-0.8/**/*.sol'",
+    "lint:ts": "yarn run --top-level eslint .",
     "make-release": "./scripts/bash/make-release.sh",
-    "verify-deployed": "./scripts/bash/verify-deployed.sh",
-    "verify-release": "./scripts/bash/verify-release.sh",
+    "migrate": "./scripts/bash/migrate.sh",
+    "prepare-contracts-and-abis-publishing": "yarn ts-node --preferTsExts ./scripts/prepare-contracts-and-abis-publishing.ts",
+    "prepare-devchain-anvil-publishing": "yarn ts-node --preferTsExts ./scripts/change-anvil-devchain-package-version.ts",
+    "revoke": "./scripts/bash/revoke.sh",
+    "set-block-gas-limit": "./scripts/bash/set_block_gas_limit.sh",
     "size:onchain": "./scripts/bash/get_smart_contract_size_from_onchain_address.sh",
     "size:artifacts": "./scripts/bash/get_smart_contract_size_from_build_artifacts.sh",
-    "ganache-dev": "./scripts/bash/ganache.sh",
-    "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
+    "sourcify-publish": "yarn ts-node ./scripts/sourcify-publish.ts",
+    "submodules:pull": "git submodule update --init --recursive",
+    "submodules:delete": "rm -rf $(git submodule | awk '{ print $2 }')",
+    "tags:compare": "./scripts/bash/compare-git-tags.sh",
+    "tags:view": "git for-each-ref 'refs/tags/core-contracts.*' --sort=-committerdate --format='%(color:magenta)%(committerdate:short) %(color:blue)%(tree) %(color:green)github.com/celo-org/celo-monorepo/releases/tag/%(color:yellow)%(refname:short)'",
```

Comment:
This change might impact documentation present in https://docs.celo.org/community/release-process/smart-contracts

So potentially this should be connected with updating docs

Reply:
Waiting to resolve my issue with GPG keys around `docs` repo access.
After that I should be able to update docs for this and other ones 🤞

Reply:
Resolved now -> here's companion PR in docs repo: https://github.com/celo-org/docs/pull/1878